src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/rest/QueryMetricsRequestHandler.java

```diff
@@ -178,6 +188,12 @@ public boolean validParams(HttpExchange exchange, List<String> metricList, List<
                 }
             }
         }
+        for (String agg: aggList) {
+            if (!agg.equals("sum") && !agg.equals("avg") && !agg.equals("min") && !agg.equals("max")) {
```

Comment:
Can we move this function MetricsDB class? That class has definitions for the supported aggregations and we can use that. In the future if we add a new aggregation like p99 the number of changes that need to be made will be minimal.